File: 프로그래머스/lv1/오하늘/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    if len(s) % 2 == 0:
        print(len(s)//2)
        n = len(s)//2
        answer = s[n-1]+s[n]
    else:
        n = len(s)//2
        answer = s[n]
        
    return answer

File: 프로그래머스/lv1/이하은/가운데 글자 가져오기.kt
class Solution {
    fun solution(s: String): String {
        val mid = s.length / 2 
        if(s.length % 2 == 0){
            return s.substring(mid - 1, mid + 1)
        }else{
            return s[mid].toString()
        }
    }
}